# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ooo/use-gcc-not-gcc33.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ooo-build-1.3.6/bin/build-ooo~	2004-11-12 12:44:40.000000000 +0100
+++ ooo-build-1.3.6/bin/build-ooo	2004-11-25 03:01:09.065129632 +0100
@@ -19,7 +19,7 @@
 
 # If we are forcing gcc33, make sure configure knows
 if test "z$FORCE_GCC33" = "zYES"; then
-CONFIGURE_OPTIONS="$CONFIGURE_OPTIONS CFLAGS= CXXFLAGS= CC=gcc33 CXX=g++33"
+CONFIGURE_OPTIONS="$CONFIGURE_OPTIONS --with-gxx-include-path=c++ CXXFLAGS=-fpermissive"
 fi
 
 ./configure $CONFIGURE_OPTIONS || exit 1;
